docs/getting-started/getting-started-with-wdl.rst

```diff
@@ -1,8 +1,4 @@
-.. role:: raw-latex(raw)
-   :format: latex
-..
-
-.. note:: This tutorial is a continuation of :doc:`Getting Started With Docker <getting-started-with-docker>`. Please complete that tutorial prior to doing this one.
+.. note:: This document assumes you have basic knowledge of what Docker is. It is recommended, but not required, to complete :doc:`Getting Started With Docker <getting-started-with-docker>` before this tutorial.
```

Comment:
'It is recommended, but not required, to complete' -> 'It is recommended, but not required, that you complete'